rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinTestUtils.java

```diff
@@ -125,4 +125,30 @@ public static byte[] flatKeysAsByteArray(List<BtcECKey> keys) {
 
         return flatPubKeys;
     }
+
+    public static void addSignaturesToInput(BtcTransaction transaction, int inputIndex, List<BtcECKey> keys) {
```

Comment:
I would like the name of this method to express that it will only sign the input if only if the input is not already signed. Wdyt?

Reply:
Something like 'Map::putIfAbsent'? Like "addSignatureIfAbsent"?.